02 Algorithm Reference/04 Importing Custom Data/04 Loading Reference Data.html

```diff
@@ -8,5 +8,10 @@
 </pre>
 <pre class="python"># If using dropbox remember to add the &amp;dl=1 to trigger a download
 file = self.Download("https://www.dropbox.com?....&amp;dl=1")
+
+# read file to a pandas DataFrame. include following imports above
```

Comment:
Let's explicitly say that the string object returned from the `Download` method should be a `csv` to use `StringIO`+`pandas.read_csv`